packages/react/src/lib/accessibility/Behaviors/Chat/chatMessageBehavior.ts

```diff
@@ -20,6 +20,7 @@ const chatMessageBehavior: Accessibility = () => ({
   attributes: {
     root: {
       [IS_FOCUSABLE_ATTRIBUTE]: true,
+      tabIndex: -1,
```

Comment:
Is this change for the prototyping or a real fix?

Reply:
it's additional safety thing. Helps both for prototype and overall behavior. Chat Message should be always focusable, meaning - to have set `tabIndex=-1`. When component re-renders based on state change, `FocusZone` won't pick it up unless you move arrow keys, so the chat message will be rendered without tabIndex - thus not focusable